vsphere/test_vsphere.py

```diff
@@ -189,7 +189,7 @@ def assertMOR(self, instance, name=None, spec=None, tags=None, count=None, subse
         if count:
             self.assertEquals(len(candidates), count)
         else:
-            self.assertTrue(len(candidates))
+            self.assertFalse(len(candidates))
```

Comment:
Why is this changed?

Reply:
the output for the method gives doesn't contain the data as expected and it will be always empty. So i had to change it to False. I am not sure how it was working before.

Reply:
Interesting, maybe the code path was never touched before.